src/components/dataDisplays/cellRenderers/DateRenderer.jsx

```diff
@@ -1,15 +1,34 @@
-import React from 'react';
+import React, { forwardRef } from 'react';
 import { get } from 'lodash-es';
+
 import { formatDate } from '../../../utils/formatters';
+import OverflowController from './OverflowController';
 import Text from '../../Text';
 
+function Core({ value, ...rest }, ref) {
+  return (
+    <Text component="span" variant="body2" ref={ref} {...rest}>
+      {value}
+    </Text>
+  );
+}
+
+const CoreForwardRef = forwardRef(Core);
+
 export default function DateRenderer({
   datum,
   accessor = 'created',
+  noWrap = false,
 }) {
-  return (
-    <Text component="span" variant="body2">
-      {formatDate(get(datum, accessor, ''))}
-    </Text>
+  const formattedValue = formatDate(get(datum, accessor, ''));
+
+  const coreComponent = <CoreForwardRef value={formattedValue} />;
```

Comment:
not really important but whenever I set a component to a variable I always capitalize that variable's name (ie CoreComponent)

Reply:
Fixed in https://github.com/WildMeOrg/codex-frontend/pull/314/commits/9ef13bfed1e6b2cff6894f501e1c2dee3cb9a077